Adamant/Modules/Chat/ViewModel/ChatViewModel.swift

```diff
@@ -262,8 +262,10 @@ final class ChatViewModel: NSObject {
                 return
             }
             
-            let isChatLoaded = await chatsProvider.isChatLoaded(with: address)
-            let isChatLoading = await chatsProvider.isChatLoading(with: address)
+            async let isChatLoadedAsync = chatsProvider.isChatLoaded(with: address)
+            async let isChatLoadingAsync = chatsProvider.isChatLoading(with: address)
+            
+            let (isChatLoaded, isChatLoading) = (await isChatLoadedAsync, await isChatLoadingAsync)
```

Comment:
I suggest to refactor this code a little bit, to avoid introduction of language keywords into the variable name:

```
async let chatState = (chatsProvider.isChatLoaded(with: address), chatsProvider.isChatLoading(with: address))
            let (isChatLoading, isChatLoaded) = await chatState
```